README.md

```diff
@@ -70,3 +70,16 @@ ctest -T memcheck
 ```
 
 Results will be in `out/Testing/Temporary/MemoryChecker.*.log`
+
+
+## Version History
+
+### Version 2.0 Release Notes
+
+#### Breaking Changes:
+- **Deprecated**: `-c` command-line option has been removed.
```

Comment:
Consider replacing `Deprecated` with `Removed` here since marking something obsolete as deprecated usually means it's just a warning about _future_ removal without removing it. Removal makes sense here since gen1 iothub connection string makes no sense in a gen2, but we shouldn't call it deprecation IMO

Reply:
Fixed